src/pages/products/products-nav-filters/products-nav-filters.js

```diff
@@ -0,0 +1,150 @@
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import Grid from '@material-ui/core/Grid';
+import { useStyles } from './products-nav-filters.styles';
+
+import {
+  setCategoryFilter,
+  setColorsFilter,
+  setModelsFilter,
+  setPatternsFilter
+} from '../../../redux/products/products.actions';
+import ProductsFilterContainer from '../../../components/products-filters-container';
+
+import ProductsNavSort from '../products-nav-sort';
+import ProductsNavSearch from '../products-nav-search';
+
+const ProductsNavFilters = () => {
+  const styles = useStyles();
+  const { filterData, filters } = useSelector(({ Products }) => ({
+    filterData: Products.filterData,
+    filters: Products.filters
+  }));
+
+  const {
+    categoryFilter,
+    colorsFilter,
+    patternsFilter,
+    modelsFilter
+  } = filters;
+
+  const categoriesNames = useMemo(
+    () => [
+      ...new Set(filterData.map(({ category }) => category.name[0].value))
+    ],
+    [filterData]
+  );
+
+  const categories = useMemo(
+    () =>
+      categoriesNames.map(
+        (category) =>
+          filterData.find(
+            ({ category: { name } }) => category === name[0].value
+          ).category
+      ),
+    [filterData, categoriesNames]
+  );
+
+  const colorsNames = useMemo(
+    () => [
+      ...new Set(filterData.map(({ colors }) => colors[0].simpleName[0].value))
+    ],
+    [filterData]
+  );
+
+  const colors = useMemo(
+    () =>
+      colorsNames.map((color) =>
+        filterData.find(({ colors }) => colors[0].simpleName[0].value === color)
+      ),
+    [filterData, colorsNames]
+  );
+
+  const patternsNames = useMemo(
+    () => [...new Set(filterData.map(({ pattern }) => pattern[0].value))],
+    [filterData]
+  );
+
+  const patterns = useMemo(
+    () =>
+      patternsNames.map((item) =>
+        filterData.find(({ pattern }) => pattern[0].value === item)
+      ),
+    [filterData, patternsNames]
+  );
+
+  const modelNames = useMemo(
+    () => [...new Set(filterData.map(({ model }) => model[0].value))],
+    [filterData]
+  );
+
+  const models = useMemo(
+    () =>
+      modelNames.map((item) =>
+        filterData.find(({ model }) => model[0].value === item)
+      ),
+    [filterData, modelNames]
+  );
+
+  const filtersOptions = {
+    categories: {
+      buttonName: 'КАТЕГОРІЇ',
```

Comment:
Please, move `buttonName` value to config

Reply:
moved